src/models/slices/location.ts

```diff
@@ -1,6 +1,15 @@
 import { LatLong, LocationBias } from '@yext/answers-core';
 
+/**
+ * The state for the user's location.
+ */
 export interface LocationState {
+  /**
+   * The geographical location bias used in a search.
```

Comment:
Can we make it clear that the `locationBias` is provided by the backend, and that `userLocation` affects search requests

Reply:
Updated!